libbeat/docs/loggingconfig.asciidoc

```diff
@@ -124,6 +125,11 @@ The number of most recent rotated log files to keep on disk. Older files are
 deleted during log rotation. The default value is 7. The `keepfiles` options has to be
 in the range of 2 to 1024 files.
 
+===== files.permissions
+
+The permission mask to apply when rotating log files. The default value is 0600. The 
+`permissions` option must be a valid file permissions mask.
```

Comment:
Hmm...I wouldn't link to wikipedia even though it's more descriptive. Linking to the golang doc might actually confuse people. 

So maybe instead of a link, you could be a bit more descriptive. Maybe say something like, "The `permissions` options must be a valid Unix-style file permissions mask expressed in octal notation (for example, `0754`). 

Or you could link to the following website, which is super useful, but I hesitate to point to it as a resource because I don't know who maintains it or how long it will be around: https://www.filepermissions.com/